docs/how-to/talos.md

```diff
@@ -20,14 +20,19 @@ This can be achieved by updating the machine config:
 machine:
   install:
     extensions:
-      - image: ghcr.io/siderolabs/drbd:9.2.0-v1.3.5
+      - image: ghcr.io/siderolabs/drbd:9.2.0-v1.3.6
+  kernel:
+    modules:
+      - name: drbd
```

Comment:
Can one add some parameters here? If so, the kernel module should be loaded with `usermode_helper=disabled`

Reply:
done: 431f26bdd43599913c57b8ec1379a50a85c8619e